stats/opentelemetry/opentelemetry.go

```diff
@@ -126,6 +130,13 @@ type MetricsOptions struct {
 	// grpc.StaticMethodCallOption as a call option into Invoke or NewStream.
 	// This only applies for server side metrics.
 	MethodAttributeFilter func(string) bool
+
+	// OptionalLabels are labels received from xDS that this component should
```

Comment:
Maybe this shouldn't mention xDS?  "labels received from LB policies"?

Reply:
Done. Yeah xDS Configures LB Policies, but LB Policies pass it to this.